### PR TITLE
Updates plugin install instructions.

### LINKED ISSUE #729: Plugins installation content is incorrect
## This is an issue with:

- [ ] Bug (site functionality or styling)
- [ ] Errata (fix needed for doc content)
- [ ] New content (guide wanted)
- [x] Update (Add missing or refresh existing content)

## Expected Behavior
Plugins reference content should reference the `sensu-install` tool, not installing via gem.

## Current Behavior
https://docs.sensu.io/plugins/1.0/installation/#gems points to using gems

## Possible Solution
ALL TEH UPDATING

### PR DESCRIPTION
**Pull request by asachs01**
## Description
Fixes #729 

## Motivation and Context
Provide clearer instructions for installing plugins.

## Review Instructions
Run via `yarn server` to confirm this renders properly. Read over new instructions to ensure they are clear.

### CODE REVIEW

**Review by rgeniesse — APPROVED** [review]
LGTM.

File: content/plugins/1.0/installation.md [hunk 1]
```diff
@@ -10,18 +10,28 @@ menu: "plugins-1.0"
...
 **Standard Installation**
 
 ```
-gem install sensu-plugins-disk-checks
+sensu-install -p disk-checks
 ```
 
+For further details, please see the [plugin installation guide][1].
+
+### Gems
```
Comment (apaskulin) [inline content/plugins/1.0/installation.md:23]: This heading seems like a duplicate of the one in line 11. Looks like maybe the heading in line 11 should be removed?

File: content/plugins/1.0/installation.md [hunk 2]
```diff
@@ -10,18 +10,28 @@ menu: "plugins-1.0"
 
 ### Gems
 
-Installing via gem is the preferred method if one exists.
-
-For production deployments make sure you [pin your gems](https://github.com/sensu-plugins/community/blob/master/best_practices/production_deployments/plugins/PINNING_VERSIONS.md)
+Installing via the built-in `sensu-install` is the preferred method of installing a plugin.
```
Comment (apaskulin) [inline content/plugins/1.0/installation.md]: Needs another word after `sensu-install`: command? tool?

File: content/plugins/1.0/installation.md [hunk 3]
```diff
@@ -10,18 +10,28 @@ menu: "plugins-1.0"
...
+
+If you're not able to install a plugin via `sensu-install`, you may use the `gem` command from Sensu's embedded ruby. 
+
+For production deployments make sure you [pin your gems][2].
+
 _NOTE: If the gem has an alpha tag then you will need to use the `--prerelease` flag or the gem will not be found._
 
+**Using Gems**
+
+`/opt/sensu/embedded/bin/gem install sensu-plugins-disk-checks`
```
Comment (apaskulin) [inline content/plugins/1.0/installation.md]:
```suggestion
{{< highlight shell >}}
/opt/sensu/embedded/bin/gem install sensu-plugins-disk-checks{{< /highlight >}}
```

**Review by apaskulin — COMMENTED** [review]
Nice work. Left a few comments, plus trying out new fancy GitHub features.

**Review by apaskulin — APPROVED** [review]

**Pull request merged**